src/commands/ProxyController.php

```diff
@@ -142,6 +148,10 @@ public function optionAliases()
      */
     public function actionSync()
     {
+        if ($this->db === null) {
+            $this->db = $this->module->proxyConnectionName;
```

Comment:
This command is running on the client side, so we should use the command --db= option only, instead of the module option. The module property is only for the server, i assumed?

Reply:
My thought was that you can configure a default connection to spare the parameter. But I think this not needed.